tests/e2e/creation_test.go

```diff
@@ -302,17 +303,21 @@ func (tc *testContext) testAllApplicationCreation(t *testing.T) error { //nolint
 }
 
 func (tc *testContext) testApplicationCreation(component components.ComponentInterface) error {
-	err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
+	err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, true, func(ctx context.Context) (bool, error) {
 		// TODO: see if checking deployment is a good test, CF does not create deployment
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
 			LabelSelector: labels.ODH.Component(component.GetComponentName()),
 		})
 		if err != nil {
-			log.Printf("error listing application deployments :%v. Trying again...", err)
-
-			return false, fmt.Errorf("error listing application deployments :%w. Trying again", err)
+			log.Printf("error listing application deployments :%v", err)
+			return false, fmt.Errorf("error listing application deployments :%w", err)
 		}
 		if len(appList.Items) != 0 {
+			if component.GetManagementState() != operatorv1.Managed {
+				// deployment exists for disabled component
+				return false, nil
+			}
+
```

Comment:
below line 324,  should we just return it?
e.g
```
allAppDeploymentsReady := true
for _, deployment := range appList.Items {
    if deployment.Status.ReadyReplicas < 1 {
        // allAppDeploymentsReady = false
        log.Printf("waiting for application deployments to be in Ready state.")
		return false, nil
    }
}
```

Reply:
Agree